metashare/api/models.py

```diff
@@ -460,6 +498,14 @@ class Task(
     def __str__(self):
         return self.name
 
+    def save(self, *args, originating_user_id=None, **kwargs):
+        ret = super().save(*args, **kwargs)
+        # To update the project's status:
+        if self.project.should_update_status():
+            self.project.notify_changed(originating_user_id=originating_user_id)
```

Comment:
After thinking about this more, I'm not sure we want an `originating_user_id` for a secondary effect like this.